### PR TITLE
Ck 49 docker image push

### PR DESCRIPTION
**Pull request by Ahsen19**
Added docker-build-and-push.yml which will build:

1. both our front and back end images
2. push these images to dockerhub

Also updated our repo link in our readme

### CODE REVIEW

File: .github/workflows/docker-build-and-push.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Docker Compose Actions Workflow
+run-name: Docker build & push
+
+on:
+  push:
+    branches: [CK-49-docker-image-push, develop]
```
Comment (JoshuaDueck) [inline .github/workflows/docker-build-and-push.yml:6]: Shouldn't we only be pushing to docker on merges/pushes to `main`?

Comment (JoshuaDueck) [reply .github/workflows/docker-build-and-push.yml]: I see you changed the line, but I still think it should be only on pushes to main, not develop, unless you want to push a staging/unstable build...

Comment (Ahsen19) [reply .github/workflows/docker-build-and-push.yml]: Main and what about develop?

Comment (JoshuaDueck) [reply .github/workflows/docker-build-and-push.yml]: Depends if you want to do an unstable build, but I think we should just start with considering our docker image to be "prod"

Comment (Ahsen19) [reply .github/workflows/docker-build-and-push.yml]: Ill change that

Comment (Ahsen19) [reply .github/workflows/docker-build-and-push.yml]: Done
Thanks for the feedback

**Review by JoshuaDueck — APPROVED** [review]
This is great, you're a champ!

**Pull request merged**